### PR TITLE
 Add hindent and brittany beautifiers for haskell

### PR DESCRIPTION
**Pull request by Tehnix**
### What does this implement/fix? Explain your changes.
This commit adds support for the hindent and brittany beautifiers for haskell.

### Does this close any currently open issues?
None outstanding in this repo, but related are https://github.com/lspitzner/brittany/issues/62.

### Any other comments?

None.

### Checklist

Check all those that are applicable and complete.

- [x] Merged with latest `master` branch
- [x] Regenerate documentation with `npm run docs`
- [x] Add change details to `CHANGELOG.md` under "Next" section
- [x] Added examples for testing to [examples/ directory](examples/)
- [x] Travis CI passes (Mac support)
- [x] AppVeyor passes (Windows support)

### CODE REVIEW

**Comment by Tehnix** [comment]
Seems travis fails on something unrelated? 

```
The command "gem install rubocop" failed and exited with 1 during
```

**Comment by lierdakil** [comment]
@Glavin001, I'd love to see this merged. Any reason not to?

File: src/beautifiers/brittany.coffee [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Brittany extends Beautifier
+  name: "brittany"
+  link: "https://github.com/lspitzner/brittany"
+  isPreInstalled: false
+
+  options: {
+    Haskell: true
```
Comment (Glavin001) [inline src/beautifiers/brittany.coffee]: `true` indicates all options are supported. Looking at code below, this should be `false` indicating language is supported with no supported options.

File: src/beautifiers/hindent.coffee [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class Hindent extends Beautifier
+  name: "hindent"
+  link: "https://github.com/commercialhaskell/hindent"
+  isPreInstalled: false
+
+  options: {
+    Haskell: true
```
Comment (Glavin001) [inline src/beautifiers/hindent.coffee]: `true` indicates all options are supported. Looking at code below, this should be `false` indicating language is supported with no supported options.

File: src/options.json [hunk 3]
```diff
@@ -2652,9 +2654,11 @@
         "title": "Default Beautifier",
         "order": -2,
         "type": "string",
-        "default": "stylish-haskell",
+        "default": "brittany",
```
Comment (Glavin001) [inline src/options.json]: This should show `stylish-haskell`.

**Comment by Glavin001** [comment]
Thank you for contributing!

**Comment by Glavin001** [comment]
Published to v0.32.0

**Pull request merged**